6. Developer Guides/REST API/Channels/info.md

```diff
@@ -12,7 +12,8 @@ Retrieves the information about the channel.
 ## Query Parameters
 | Argument | Example | Required | Description |
 | :--- | :--- | :--- | :--- |
-| `roomId` | `ByehQjC44FwMeiLbX` | Required | The channel's id |
+| `roomId` | `ByehQjC44FwMeiLbX` | Required (if no roomName) | The channel's id |
+| `roomName` | `ByehQjC44FwMeiLbX` | Required (if no roomId) | The channel's name |
```

Comment:
Can you change the example to be like a room name? This applies to all of these, thanks for the pull request!